### DISCUSSION TITLE
Edit toots

### DISCUSSION THREAD

**Discussion by NickColley** [NickColley/semaphore#20] [New features and ideas]
Continuation of https://github.com/nolanlawson/pinafore/issues/2179#issuecomment-1339261269

Make it easy to see toots that have been edited, their history and also edit our own toots within Semaphore.

**👉🏻  [View progress of this feature](https://github.com/semaphore-social/semaphore/milestone/2)**

**Comment by strypey** [comment]
This is perhaps the highest priority item for me, as a longtime Pinafore user who was thrilled to recently discover Semaphore (thanks so much for doing this)! I often have to shift from using Semaphore on a laptop back to FediLab to edit a post.

FWIW fediverse posts were only ever called "toots" in Mastodon, and even they've dropped it from their UI now.
